pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -661,7 +661,8 @@ func (cp *capacityPlugin) checkHierarchicalQueue(attr *queueAttr) error {
 		}
 		// Check if the parent queue's capability is less than the child queue's capability
 		if attr.capability.LessPartly(childAttr.capability, api.Zero) {
-			return fmt.Errorf("queue <%s> capability is less than its child queue <%s>", attr.name, childAttr.name)
+			return fmt.Errorf("queue <%s> capability <%s> is less than its child queue <%s> capability <%s>",
+				attr.name, attr.capability.String(), childAttr.name, childAttr.capability)
```

Comment:
Let’s unify the writing style. Don’t call String() in attr.capability is better :)